src/utils.ts

```diff
@@ -1,6 +1,9 @@
 "use strict";
 
-var addressCodec = require("ripple-address-codec");
+import { createHash } from "crypto";
```

Comment:
I'm not sure if "crypto" is available on all the platforms that this code will run on? (I think it's a Node.js module.) In ripple-lib, I believe we use [create-hash](https://github.com/crypto-browserify/createHash). Just a thought.

Reply:
It is a node.js module but it should be polyfilled by webpack for the distribution.

Reply:
I'd rather rely on a core module or a webpack polyfill than a public library. 

It looks like ripple-lib uses the core crypto module here: https://github.com/ripple/ripple-lib/blob/develop/src/common/hashes/sha512Half.ts#L1